apps/storybook/src/SearchBox.stories.tsx

```diff
@@ -0,0 +1,127 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, StoryFn } from '@storybook/react';
+import React from 'react';
+import {
+  SearchBox,
+  SearchBoxProps,
+  IconButton,
+  Text,
+  Flex,
+  Divider,
+} from '@itwin/itwinui-react';
+import { SvgCaretDownSmall, SvgCaretUpSmall } from '@itwin/itwinui-icons-react';
+
+export default {
+  component: SearchBox,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+  },
+  title: 'Input/SearchBox',
+} as Meta<SearchBoxProps>;
+
+export const Basic: StoryFn<SearchBoxProps> = () => {
+  return (
+    <Flex flexDirection='column'>
+      <SearchBox disabled inputProps={{ placeholder: 'Basic search' }} />
```

Comment:
can we make basic example actually basic? it's showing too much right now

Reply:
Updated stories